### PR TITLE
Readme update - remove *condition for logs

### PR DESCRIPTION
**Pull request by vishweshbankwar**
`1.6.0` release is out. It contains stable otlp log exporter.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,15 +21,10 @@ files.
 
 | Signal  | Status     |
 | ------- | ---------- |
-| Logs    | Stable*    |
+| Logs    | Stable     |
```
Comment (reyang) [inline README.md:24]: This is great! Can we replace the table with a simple statement "stable across logs/metrics/traces"?

Comment (vishweshbankwar) [reply README.md]: updated: is this better?

**Review by reyang — APPROVED** [review]

**Review by cijothomas — APPROVED** [review]
🎉 :100:

**Comment by cijothomas** [comment]
https://opentelemetry.io/docs/instrumentation/net/#status-and-releases @vishweshbankwar this too please!

**Review by utpilla — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -19,16 +19,7 @@ files.
...
-| ------- | ---------- |
-| Logs    | Stable*    |
-| Metrics | Stable     |
-| Traces  | Stable     |
-
-*While the `OpenTelemetryLoggerProvider` (i.e integration with `ILogger`) is
- stable, the [OTLP Exporter for
- Logs](https://github.com/open-telemetry/opentelemetry-dotnet/tree/main/src/OpenTelemetry.Exporter.OpenTelemetryProtocol#enable-log-exporter)
- is still non-stable.
+**Stable** across all 3 signals i.e. `logs`, `metrics` and `traces`.
```
Comment (utpilla) [inline README.md:22]:
```suggestion
**Stable** across all 3 signals i.e. `logs`, `metrics`, and `traces`.
```

Comment (utpilla) [inline README.md:22]: Do we want to use capital letters?
```suggestion
**Stable** across all 3 signals i.e. `Logs`, `Metrics`, and `Traces`.
```

Comment (vishweshbankwar) [reply README.md:22]: You made me look this up 😆 
https://www.grammarly.com/blog/comma/

Comment (utpilla) [reply README.md]: Haha yes, the infamous Oxford comma.

**Review by utpilla — APPROVED** [review]

**Pull request merged**